### PR TITLE
feat: Nft details inventory tab

### LINKED ISSUE #146: NFT details inventory tab
A new tab called "Inventory" should be visible in the NFT details page with the following paginated table:

| Token ID | Owner |
|--------|--------|
| 1 | ak_2QkttUg...Yd |

[Example NFT without templates](https://mainnet.aeternity.io/mdw/v2/aex141/ct_2w7MHbcyk5iM4yijZmjGcUF2DSZuVM1Mueppx5SSvPn5cgLJuZ/owners).

If the NFT has `mintable_templates` extension a different paginated table should be displayed with the following columns:

| Template ID | Supply | Limit | Create Transaction |
|--------|--------|--------|--------|
| 8 | 8 | 8 | th_ojhH9bD...z4 | 

[Example NFT with templates](https://testnet.aeternity.io/mdw/v2/aex141/ct_2oq4kSd4j1VkkbupueXLdHwYEJdY8Ntzvp1FFkMB1gYyXkYPcV).
[Example templates](https://testnet.aeternity.io/mdw/v2/aex141/ct_2oq4kSd4j1VkkbupueXLdHwYEJdY8Ntzvp1FFkMB1gYyXkYPcV/templates).

When a template is expanded the user should be able to see NFTs inside a template and the table should contain:
| Token ID | Owner | Create Transaction |
|--------|--------|--------|
| 36 | ak_8Ujt76Q...YN | th_NxjFwCY...ot | 

[Example tokens in a NFT template](https://testnet.aeternity.io/mdw/v2/aex141/ct_2oq4kSd4j1VkkbupueXLdHwYEJdY8Ntzvp1FFkMB1gYyXkYPcV/templates/8/tokens).

### PR DESCRIPTION
**Pull request by janmichek**
## Description
resolves #146 

followup for counters https://github.com/aeternity/aescan/issues/496

## Demo

**Case NFT with templates**
testnet ct_2oq4kSd4j1VkkbupueXLdHwYEJdY8Ntzvp1FFkMB1gYyXkYPcV
image

**Case NFT without templates (direct owners display)**
mainnet ct_2w7MHbcyk5iM4yijZmjGcUF2DSZuVM1Mueppx5SSvPn5cgLJuZ
image

**Case NFT with templates and owners** 
(no data)

## Checklist:

- [x] I have read and followed the [Contributing Guide](../../CONTRIBUTING.md)  
- [x] display templates and owners

### CODE REVIEW

File: src/components/NftsDetailsPanel.vue [hunk 1]
```diff
@@ -1,65 +1,65 @@
 <template>
-  <app-panel class="nft-details-panel">
+  <app-panel class="nfts-details-panel">
```
Comment (janmichek) [inline src/components/NftsDetailsPanel.vue:2]: refactor naming according to convention, made a mistake before

File: src/components/NftsPanel.vue [hunk 2]
```diff
@@ -42,9 +42,7 @@ async function loadNextNfts() {
 }
 
 if (process.client) {
-  await fetchNfts(
-    `/v2/aex141?limit=${limit}`,
-  )
+  await fetchNfts({ queryParameters: `/v2/aex141?limit=${limit.value}` })
```
Comment (janmichek) [inline src/components/NftsPanel.vue:45]: also fix from previous part

File: src/components/NftsTemplatesOwnersTableCondensed.vue [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+<style scoped>
+.nfts-templates-owners-table-condensed {
+  &__table {
+    padding: 0 var(--space-1) var(--space-7);
+    margin-bottom: var(--space-5);
+  }
+
+  &__header {
+    font-family: var(--font-primary);
```
Comment (janmichek) [inline src/components/NftsTemplatesOwnersTableCondensed.vue:75]: header and data needed to be overstyled as they are tables within table cell. It inherited styles

**Comment by michele-franchi** [comment]
@janmichek Could you align this PR to develop?

**Comment by janmichek** [comment]
> @janmichek Could you align this PR to develop?

rebased

**Comment by michele-franchi** [comment]
Needs to be rebased again after the recent merges @janmichek

**Comment by janmichek** [comment]
@lukeromanowicz @michele-franchi all fixed, you can review now

File: src/components/NftsTransfersPanel.vue [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        :nft-transfers="nftTransfers"/>
+    </paginated-content>
+  </app-panel>
+</template>
+
+<script setup>
+import { storeToRefs } from 'pinia'
+import { computed } from 'vue'
+import { useNftDetailsStore } from '@/stores/nftDetails'
+import { isDesktop } from '~/utils/screen'
```
Comment (lukeromanowicz) [inline src/components/NftsTransfersPanel.vue:23]: weird import

File: src/components/NftsTransfersPanel.vue [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+      <nfts-transfers-table-condensed
+        class="nfts-transfers-panel__table-condensed"
+        :nft-transfers="nftTransfers"/>
+    </paginated-content>
+  </app-panel>
+</template>
+
+<script setup>
+import { storeToRefs } from 'pinia'
+import { computed } from 'vue'
```
Comment (lukeromanowicz) [inline src/components/NftsTransfersPanel.vue:21]: Computed doesn't need to be imported. It's built in

File: src/pages/nfts/[id].vue [hunk 6]
```diff
@@ -28,15 +32,21 @@ import { storeToRefs } from 'pinia'
 import { nftsHints } from '@/utils/hints/nftHints'
 import PageHeader from '@/components/PageHeader'
 import { useNftDetailsStore } from '@/stores/nftDetails'
-import NftDetailsPanel from '@/components/NftDetailsPanel'
+import NftDetailsPanel from '~/components/NftsDetailsPanel'
```
Comment (lukeromanowicz) [inline src/pages/nfts/[id].vue:35]: wrong import alias

**Review by lukeromanowicz — CHANGES_REQUESTED** [review]
Very very nice job. Close to perfect. Very impressing :sunglasses:

**Comment by janmichek** [comment]
> Very very nice job. Close to perfect. Very impressing 😎

Thanks for the great suggestions. Fixed

**Review by lukeromanowicz — APPROVED** [review]
Very nice, well done :rocket:

**Review by michele-franchi — APPROVED** [review]
Good job! I left two minor suggestions, please have a look at it.

File: src/utils/hints/nftHints.js [hunk 7]
```diff
@@ -12,4 +12,11 @@ export const nftsHints = {
   transferTokenId: 'The ID of the NFT that was transferred.',
   transferRecipient: 'The recipient of the NFT.',
   transferSender: 'The sender of the NFT.',
+  inventoryTemplateId: 'The ID of the NFT template.',
+  inventorySupply: 'The number of NFTs that are minted from this template.',
```
[the remark this answers is not in the document]
Comment (janmichek) [reply src/utils/hints/nftHints.js]: accepted

File: src/utils/hints/nftHints.js [hunk 8]
```diff
@@ -12,4 +12,11 @@ export const nftsHints = {
...
+  inventoryLimit: 'The maximum number of NFTs that can be minted from this template.',
```
[the remark this answers is not in the document]
Comment (janmichek) [reply src/utils/hints/nftHints.js]: accepted

**Pull request merged**